libvast/src/value_index.cpp

```diff
@@ -248,6 +248,8 @@ bool inspect(detail::legacy_deserializer& source, value_index_ptr& x) {
 }
 
 vast::chunk_ptr chunkify(const value_index_ptr& idx) {
+  if (!idx)
+    return nullptr;
```

Comment:
Doing this makes it impossible to serialize `value_index_ptr`s that are `nullptr`. I think I was wrong about this, sorry.

Reply:
Taken out of the review again.